R/compute_dea.R

```diff
@@ -32,6 +34,8 @@ compute_dea <- function(
 
   rts <- match.arg(rts)
   orientation <- match.arg(orientation)
+  # Allow missing argument for id
+  id <- if (missing(id)) NULL else id
```

Comment:
Best practice is to use NULL as default value for optional arguments. I suggest we do this, and then move `id` argument after the input and output arguments.

Reply:
Agreed!